grpc-proto/proto/schema.proto

```diff
@@ -25,8 +25,6 @@ package stargate;
 
 // TYPES
 
-// TABLES
-
 message UserDefinedTypeCreate {
   string keyspaceName = 1;
```

Comment:
I missed it in earlier reviews, but gRPC conventions are snake case for fields, as in `keyspace_name`. That's what we've been doing in `query.proto` so we should stay consistent.

Reply:
Agreed.

Reply:
will do

Reply:
fixed